appmonitoring/validation-helm/app-monitoring-addon/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's remember to update the release instructions to explain how to copy over the chart from AKS RP repo.
Let's also make a record in _change.log_ file describing what we did here.